target/linux/mediatek/image/filogic.mk

```diff
@@ -942,3 +942,13 @@ define Device/zyxel_nwa50ax-pro
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
 endef
 TARGET_DEVICES += zyxel_nwa50ax-pro
+
+define Device/ruijie_rg-x60-pro
+  DEVICE_VENDOR := Ruijie
+  DEVICE_MODEL := Ruijie RG X60 Pro
```

Comment:
Model usually doesn't contain vendor name. Please check the label on the device case.